lib/dor/text_extraction/ocr.rb

```diff
@@ -43,13 +43,13 @@ def cleanup
 
         # e.g. /abbyy/RESULTXML/ab123cd4567.xml.result.xml
         abbyy_results = Dor::TextExtraction::Abbyy::Results.find_latest(druid: bare_druid, logger:)
-        if abbyy_results # this could be nil if there is no latest result XML file located
+        if abbyy_results # this could be nil if there is no latest result XML file for this druid
```

Comment:
i initially had a bit of trouble parsing this comment, figuring that something was accidentally omitted after "located".  i realized how i was misreading it when giving this PR a once over after posting it... but i still think this reads just a bit more clearly?